Dockerfile

```diff
@@ -33,6 +33,17 @@ RUN uv pip install torch_geometric -f https://data.pyg.org/whl/torch-2.5.0+cu124
 COPY modules/requirements.txt requirements.txt
 RUN uv pip install -r requirements.txt
 RUN uv pip install sknw
+# Install Sentence Transformers
+RUN pip3 install sentence_transformers
```

Comment:
`sentence_transformers` is already installed from `requirements.txt`. Please remove this.